homework-submission/react-hyf/react-hyf-client/src/HomeworkWeek3/Image.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Image = (props) => {
+  return <img src={props.photo} alt={`face of ${props.name}`}/>
```

Comment:
Very literal description of the picture 😄 I think "Picture of ..." would be more fitting